mcclatchy/zones
# Zephr updates

### What does this PR do?

It adds two new type values to the zephr config object: subscriber and dma. These types will pass the value of the corresponding `locker.user` function as a string. 

### Steps to test:

1. Start a local webserver in the zones directory on port 3000
2. Unzip the [overrides.zip](https://github.com/mcclatchy/zones/files/11985366/overrides.zip) file in your overrides directory
3. Go to this article in Chrome as a logged in sub: https://www.newsobserver.com/entertainment/article277045118.html
4. In the elements panel of DevTools, do a search for `zone-taboola-recommendations`
5. Confirm that the payload from Zephr includes the id `taboola-in-market-mid-article-thumbnails`

If you would like to confirm both behaviors are working .. 

6. Go to the same page in an Incognito window (the same overrides carried over for me)
7. Confirm that the payload from Zephr includes the id `taboola-out-of-market-mid-article-thumbnails`

_I had to reload each version a second time for all the overrides to kick in._
